static/sass/styles.scss

```diff
@@ -48,11 +48,19 @@
   max-width: none;
 }
 
-.empty-state-message {
-  .icon {
-    height: 100%;
-    width: 35%;
-  }
+.u-empty-state-icon {
+  height: 2.5rem;
+  margin-bottom: 20px;
+  margin-top: 25px;
```

Comment:
We shouldn't use `px` values, but always prefer the `rem` unit or even better the `$sp-large` or alike variables from vanilla

Reply:
Gives this a read, then you might not need css for it at all: https://vanillaframework.io/docs/settings/spacing-settings